cypress.config.js

```diff
@@ -0,0 +1,21 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  env: {
+    do_reset_db: true,
+  },
+
+  e2e: {
+    baseUrl: 'http://studio.127.0.0.1.nip.io:8080',
+    //baseUrl: 'https://serve-dev.scilifelab.se',
```

Comment:
Is it possible to make this an environment variable?

Reply:
The env var CYPRESS_BASE_URL can be used to override this value as well as --env from the command line. The value specified in the cypress.config.js file will be a default value that it is easy to override depending on needs, such as in CI pipelines.